channels/ON-CHAIN.md

```diff
@@ -104,34 +104,35 @@ Serialization defined [here](../serializations.md#channel-deposit-transaction)
 
 - `channel_id`: channel id as recorded on chain
 - `from`: sender of the deposit
-- `amount`: amount of coins deposited
+- `amount`: amount of tokens deposited
 - `ttl`:
 - `fee`:
 - `nonce`: account nonce of the submitter
 
 
 ### `channel_withdraw`
 
-Channels should generally not be used to hold significant amounts of coins but
-being able to withdraw locked coins might still be of use.
+Channels should generally not be used to hold significant amounts of tokens but
+being able to withdraw locked tokens might still be of use.
 
 The `from` account MUST be a participant in the target channel. The `amount`
 MUST be less or equal than the sum of all participants balances, i.e. channels
-cannot create coins out of thin air.
-
+cannot create tokens out of thin air. The fee is paid by the `from` account and that
+account should hold enough tokens to pay the fee, i.e., the fee is subtracted before
+the withdrawn tokens arrive.
 
 Serialization defined [here](../serializations.md#channel-withdraw-transaction)
 
 
 - `channel_id`: channel id as recorded on chain
-- `from`:
-- `amount`:
+- `from`: receiver of the withdraw
```

Comment:
I see - "receiver of the withdraw" means "sender of the transaction, receiver of the withdrawn tokens".

Reply:
Well spotted, I bumped into it while testign as well. Update on its way.